### ISSUE TITLE
Sddm: how to make sure session started is not Plasma, not Plasma Debug?

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-issues-old#192]
it seems that we run Plasma Debug when looking at ksysguard:
![Screenshot_20190422_182849](https://user-images.githubusercontent.com/840070/56511351-a607b480-652c-11e9-9d8a-d4a5737383d6.png)

How can we make sure Plasma is started instead of Debug?

**Comment by NuLogicSystems** [comment]
I already fixed this in git.
https://github.com/netrunner-common/netrunner-settings-sddm/commit/a9d660587ce1e2e81c5348398aa9d05f165ba383

**Comment by star-buck** [comment]
but this is autologin?

**Comment by star-buck** [comment]
i think for next release we could switch back to Breeze default sddm theme, if the custom background we set does stay after an update of Plasma/sddm.

**Comment by NuLogicSystems** [comment]
Yes, but even though It's in the autologin section, it does set the global default none the less.
The upstream default if this line is blank is plasma-debug.
It's the .desktop extension that causes sddm to ignore this line.
You can see this in the sddm-kcm by switching back and forth with and without the .desktop extension in the sddm.conf.

**Comment by star-buck** [comment]
okay, so i think we solved this in Debian before, not sure how though.

**Comment by NuLogicSystems** [comment]
Debian's version of sddm might not have the .desktop extension issue, yet.

**Comment by ScarlettGatelyMoore** [comment]
I don't seem to have that view in ksysguard :( but at a glance in the latest debian next iso this seems resolved? 
scarlett@scarlett-netrunner:~$ ps aux | grep startkde
root       608  0.0  0.3  62544 14472 ?        S    11:43   0:00 /usr/lib/x86_64-linux-gnu/sddm/sddm-helper --socket /tmp/sddm-authba6e695b-f347-4bf9-bfbc-d85ef1ea548a --id 1 --start /usr/bin/startkde --user scarlett

**Comment by NuLogicSystems** [comment]
Yes, turns out that the plasma-debug.desktop file was something we packaged and shipped.
https://github.com/ds9-common/default-settings-session-debug
Removal of this package also fixes the issue. However, it still doesn't negate the issue of the way sddm parses it's config files, and/or legacy file, at least not on arch/manjaro.
